streams/kr.m3u

```diff
@@ -53,30 +55,78 @@ http://mediaworks.dema.mil.kr:1935/live_edge/cudo.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="KETV.kr" status="online",KETV (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/wowtv_540/playlist.m3u8
 #EXTINF:-1 tvg-id="KNNTV.kr" status="online",SBS KNN (450p) [Not 24/7]
 http://211.220.195.200:1935/live/mp4:KnnTV.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="KPlus.sg" status="error",K+ (576p) [Geo-blocked]
 http://210.210.155.35/uq2663/h/h08/index.m3u8
+#EXTINF:-1 tvg-id="MBCSportsPlus.kr" status="online",MBC Sports+ (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/mbc_sports_540/playlist.m3u8
+#EXTINF:-1 tvg-id="MBCTV.kr" status="online",MBC (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/mbc_540/playlist.m3u8
 #EXTINF:-1 tvg-id="MBCTV.kr" status="error",MBC (1080p)
 http://123.254.72.24:1935/tvlive/livestream2/playlist.m3u8
+#EXTINF:-1 tvg-id="MBN.kr" status="online",MBN (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/mbn_540/playlist.m3u8
+#EXTINF:-1 tvg-id="Mnet.kr" status="online",MNet (576p) [Not 24/7]
+http://51.81.66.195:1935/krtv/mnet_540/playlist.m3u8
 #EXTINF:-1 tvg-id="MTN.kr" status="online",MTN (1080p) [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UClErHbdZKUnD1NyIUeQWvuQ/live
 #EXTINF:-1 tvg-id="MTN.kr" status="online",MTN (720p) [Not 24/7]
 http://183.110.27.87/mtnlive/720/playlist.m3u8
 #EXTINF:-1 tvg-id="NBS.kr" status="online",NBS Korea Agricultural Broadcasting (720p)
 https://media.joycorp.co.kr:4443/live/ngrp:live_all/playlist.m3u8
+#EXTINF:-1 tvg-id="OCN.kr" status="online",OCN (576p) [Not 24/7]
+http://51.81.66.195:1935/krtv/ocn_540/playlist.m3u8
+#EXTINF:-1 tvg-id="OCNMovies.kr" status="online",OCN Movies (576p) [Not 24/7]
+http://51.81.66.195:1935/krtv/cgv_540/playlist.m3u8
+#EXTINF:-1 tvg-id="SBSGolf.kr" status="online",SBS Golf (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/sbs_golf_540/playlist.m3u8
+#EXTINF:-1 tvg-id="SBSSports.kr" status="online",SBS Sports (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/sbs_sports_540/playlist.m3u8
+#EXTINF:-1 tvg-id="SBSTV.kr" status="online",SBS (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/sbs_540/playlist.m3u8
+#EXTINF:-1 tvg-id="SkySports.kr" status="online",Sky Sports (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/sky_sports_540/playlist.m3u8
+#EXTINF:-1 tvg-id="SPOTV.kr" status="online",SPOTV (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/spotv_540/playlist.m3u8
+#EXTINF:-1 tvg-id="SPOTV2.kr" status="online",SPOTV2 (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/spotv_2_540/playlist.m3u8
+#EXTINF:-1 tvg-id="TVChosun.kr" status="online",TV Chosun (540p) [Not 24/7]
+http://51.81.66.195:1935/krtv/tvchosun_540/playlist.m3u8
 #EXTINF:-1 tvg-id="TBSTV.kr" status="online",TBS Seoul (720p)
 https://cdntv.tbs.seoul.kr/tbs/tbs_tv_web.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Tooniverse.kr" status="online",Tooniverse (576p) [Not 24/7]
+http://51.81.66.195:1935/krtv/tooniverse_540/playlist.m3u8
+#EXTINF:-1 tvg-id="tvN.kr" status="online",tvN (576p) [Not 24/7]
+http://51.81.66.195:1935/krtv/tvn_540/playlist.m3u8
```

Comment:
DMCAd

Reply:
TvN

Reply:
https://github.com/iptv-org/iptv/issues/1831 this?

Reply:
the tvn I added is not from Nagravision, it's from CJ E&M

Reply:
https://en.wikipedia.org/wiki/TvN_(Asian_TV_channel)

Reply:
Well they have the rights to "encrypt" their stream. While the channel is from CJ E&M, Nagravision is the society that provides encryption for those channels, in partnership with them.

Reply:
cj e&m never uploaded any dmca to github and i didn't find any dmca from companies representing cj e&m

Reply:
in this pr there are more CJ E&M channels